src/pydot/core.py

```diff
@@ -618,6 +659,31 @@ def get_sequence(self):
         """Get sequence"""
         return self.obj_dict["sequence"]
 
+    @staticmethod
+    def _format_attr(key: str, value):
+        """Turn a key-value pair into an attribute, properly quoted."""
+        if value == "":
+            value = '""'
+        if value is not None:
+            return f"{key}={quote_attr_if_necessary(value)}"
+        return key
+
+    def formatted_attr_list(self):
+        """Return a list of the class's attributes as formatted strings."""
+        return [
+            self._format_attr(k, v)
+            for k, v in self.obj_dict["attributes"].items()
+        ]
+
+    def attrs_string(self, prefix=""):
+        """Format the current attributes list for output.
+
+        The `prefix` string will be prepended iff text is generated."""
```

Comment:
```suggestion
        The `prefix` string will be prepended if text is generated."""
```

Reply:
Heh. I actually did mean "iff" here, a shorthand for "if and only if". But that's probably a bit jargony, so I should write it out.

Reply:
Fixed to be less fake-clever.